core/src/test/java/org/infinispan/api/ConditionalOperationsConcurrentTest.java

```diff
@@ -132,13 +135,13 @@ protected void testOnCaches(List<Cache> caches, CacheOperation operation) {
       assertFalse(failureMessage, failed.get());
    }
 
-   private String[] generateValidMoves() {
-      String[] validMoves = new String[operations];
+   @BeforeTest
+   public void generateValidMoves() {
+      validMoves = new String[operations];
```

Comment:
`operations` is initialized in the constructor, you could also initialize `validMoves` the same way.